pkgs/top-level/python-packages.nix

```diff
@@ -3394,14 +3394,14 @@ in {
   pytest-rerunfailures = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Thanks, updated 👍